build.sbt

```diff
@@ -2,7 +2,7 @@ name := "ibm-tdml-processor"
  
 organization := "io.github.openDFDL"
  
-version := "0.1.0-SNAPSHOT"
+version := "0.1.0"
```

Comment:
Should crossScalaVersions below be changed, now that we're not supporting 2.11 anymore?

Reply:
Yep. If anyone tries to test with Scala 2.11 they'll fail to resolve the dependencies--as of Daffodil 3.0.0 we no longer publish daffodil jars with Scala 2.11.